### ISSUE TITLE
Visdom plot do not update after 1800 iterations

### ISSUE THREAD

**Issue by gszh** [facebookresearch/visdom#665]
When I show the losses on visdom, my plot do not update after 1800 iterations. Can anyone help me with this? Thank you. 

![image](https://user-images.githubusercontent.com/12854162/64086663-21dffc00-cd6c-11e9-8b8a-917b5677175b.png)

My codes are here:

```python
def update_vis_plot(iteration, loss1, loss2, loss3, window1, window2, update_type,
                    epoch_size=1):
    viz.line(
        X=torch.ones((1, 4)).cpu() * iteration,
        Y=torch.Tensor([loss1, loss2, loss3, loss1 + loss2 + loss3]).unsqueeze(0).cpu() / epoch_size,
        win=window1,
        update=update_type
    )
```
The update type is append.

```python
update_vis_plot(iteration, loss1.item(), loss2.item(), loss3.item(),
                                    iter_plot, epoch_plot, 'append')
```

**Comment by JackUrb** [comment]
Hi @gszh - are you sure nothing is being plotted, or is it just that the zoom has stopped updating? You can try using the plotly interface controls on the frontend or set axis bounds with the python client as shown in the demo.

**Comment by gszh** [comment]
Sorry for not replying sooner. My zoom just stopped updating. I will try to set axis bounds and see how it goes.

**Comment by JackUrb** [comment]
Did you manually zoom into the plot at any point? #522 was introduced so that zoom would be retained even when new data was appended because it was frustrating behavior to lose manual zoom settings when zoomed in on a live graph.
